frontend/public/style/_theme-dark.scss

```
@@ -0,0 +1,19 @@
+:root {
+  --co-global--palette--blue-600: var(--pf-global--palette--blue-600);
+  --co-global--palette--purple-600: var(--pf-global--palette--purple-600);
+  --co-global--palette--gold-700: var(--pf-global--palette--gold-700);
+  --co-global--palette--purple-700: var(--pf-global--palette--purple-700);
+
+  // dark
+  --co-global--dark--palette--blue-600: var(--pf-global--palette--blue-200);
+  --co-global--dark--palette--purple-600: var(--pf-global--palette--purple-200);
+  --co-global--dark--palette--gold-700: var(--pf-global--palette--gold-200);
```

Related to above suggestion.
```suggestion
  --co-global--dark--palette--orange-700: var(--pf-global--palette--orange-200);
```